pkg/api/bytes_test.go

```diff
@@ -57,6 +58,21 @@ func TestBytes(t *testing.T) {
 		}
 	})
 
+	t.Run("download-with-targets", func(t *testing.T) {
+		resp := request(t, client, http.MethodGet, resource+"/"+expHash+"?targets=0x222", nil, http.StatusOK)
+		data, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		if !bytes.Equal(data, content) {
+			t.Fatalf("data mismatch. got %s, want %s", string(data), string(content))
+		}
+		if resp.Header.Get("Targets") != targets {
+			t.Fatalf("targets mismatch. got %s, want %s", resp.Header.Get("Targets"), targets)
+		}
+	})
```

Comment:
since the bytes download is already being tested separately, and this feature is about obtaining targets from a query parameter, i suggest the following:

```suggestion
	t.Run("download-with-targets", func(t *testing.T) {
		resp := request(t, client, http.MethodGet, resource+"/"+expHash+"?targets="+targets, nil, http.StatusOK)
		if resp.Header.Get("Targets") != targets {
			t.Fatalf("targets mismatch. got %s, want %s", resp.Header.Get("Targets"), targets)
		}
	})
```

i really don't see a need to download the chunk, let alone check that its content is correct (this is covered by `t.Run("download"...` anyways)

LMK what you think

Reply:
No need for verification, I will change it to this.